atomic

```diff
@@ -119,6 +116,15 @@ if __name__ == '__main__':
     parser.add_argument('--debug', default=False, action='store_true')
     subparser = parser.add_subparsers(help=_("commands"))
 
+    # atomic cluster (if available)
+    try:
+        import commctl.cli
+        clusterp = subparser.add_parser(
```

Comment:
Yes that is true, but that is more of a case where we are differentiating from normal OS and Atomic Host.

This cluster command will be available on the normal OS correct?  IE RHEL as well as RHEL atomic host?